resources/event-sources/templates/100-pre-upgrade-job.yaml

```diff
@@ -0,0 +1,97 @@
+# This job is a kyma custom resource.
+# Please take caution during upgrade to not to remove it
```

Comment:
Why `helm del --purge <>` is not apt here?

The problem with this approach is: if I do `helm ls --all`, `event-bus` is still listed here. Then taking out the `event-bus` chart in the next release will be difficult cause if a chart is missing from resources but is installed, kyma-installer does not uninstall the chart.

Reply:
Done